addon/mixins/authenticated-route-mixin.js

```diff
@@ -78,7 +81,15 @@ export default Mixin.create({
       let authenticationRoute = this.get('authenticationRoute');
       assert('The route configured as Configuration.authenticationRoute cannot implement the AuthenticatedRouteMixin mixin as that leads to an infinite transitioning loop!', this.get('routeName') !== authenticationRoute);
 
-      if (!this.get('_isFastBoot')) {
+      if (this.get('_isFastBoot')) {
+        const fastboot = getOwner(this).lookup('service:fastboot');
+        const cookies = getOwner(this).lookup('service:cookies');
+
+        cookies.write('ember_simple_auth-redirectTarget', transition.intent.url, {
+          path: '/',
+          secure: fastboot.get('request.protocol') === 'https'
+        });
+      } else {
```

Comment:
Those came in as part of the rebase. They were changes on master after I had branched. There are no more changes for me to rebase. What am I missing? The failures seem to be related to a phantomjs install problem for Ember 1.12.